### ISSUE TITLE
in

### ISSUE THREAD

**Issue by langbiantianya** [Kotlin/dataframe#422]

**Comment by langbiantianya** [comment]
The version used is 0.11.0

**Comment by zaleslaw** [comment]
@langbiantianya please answer or provide an example of how to reproduce it?

**Comment by langbiantianya** [comment]
Generated a ktor 2.3.2 using IDEA, Gradle version is 7.4 Introduced dataframe version 0.10.1 in it, used workbook to parse it, the rest is the same as in the docs, the same way it can parse xlsx but can't parse xls. to solve the problem I had to use a poi to get around it.

**Comment by langbiantianya** [comment]
The same problem occurs in 0.11

**Comment by langbiantianya** [comment]
The problem only occurs after packing

**Comment by vishwanjalijadhav** [comment]
Do we have any workaround for this? I am facing same issue.

**Comment by Jolanrensen** [comment]
@vishwanjalijadhav so the problem is that it complains about not having access to "poi-ooxml" after packaging `dataframe-excel` in a library of yours?

Could you try manually adding the dependencies:
`api("org.apache.poi:poi-ooxml:5.3.0")`
(and potentially)
`api("org.apache.poi:poi:5.3.0")`

DataFrame already shares `poi` as `api()` dependency, but `poi-ooxml` is added as `implementation` for the moment. If this workaround works for you, we can probably fix it on our side.

**Comment by zaleslaw** [comment]
@Allex-Nik please reproduce this issue

**Comment by Allex-Nik** [comment]
Reproduced here: https://github.com/Allex-Nik/xls-parsing-issue.git

**Comment by Jolanrensen** [comment]
So, this likely can be solved by updating our documentation with a workaround on the user side?

**Comment by Allex-Nik** [comment]
@Jolanrensen Yes, I am updating our documentation and will enrich the error message
